### PR TITLE
UAT Updates. Updates to the popover component so that is supports pro…

### PR DESCRIPTION
**Pull request by bryancodazen**
…grammatic opening upon load. Updated textures so they dont block game items (labels, units, centers). Updated icons to have consistent drop shadows. Added svg options interface to the web dip theme and added a drop shadow.

Addresses UAT Numbers: 1,3,4

### Testing

1. Spool up local env or run node dev server
2. Verify UAT numbers 1,3, and 4 are complete. 

UAT sheet: https://docs.google.com/spreadsheets/d/1Z_S5REYlbU0lcicvk-Ou_9z-ncS6ySnL61YMqfK-W0s/edit#gid=0

### CODE REVIEW

File: beta-src/src/components/svgr-components/WDActionIcon.tsx [hunk 1]
```diff
@@ -1,51 +1,28 @@
...
 import { navIconProps } from "../../interfaces/Icons";
 import UIState from "../../enums/UIState";
 
 const WDActionIcon: React.FC<navIconProps> = function ({
   iconState = UIState.INACTIVE,
 }): React.ReactElement {
+  const theme = useTheme();
+  console.log({
+    theme,
+  });
```
Comment (kenneth-yf-wang) [inline beta-src/src/components/svgr-components/WDActionIcon.tsx:12]: should this console.log be removed?

**Review by kenneth-yf-wang — APPROVED** [review]
overall LGTM!

File: beta-src/src/components/svgr-components/WDActionIcon.tsx (hunk 1 above)
Comment (bryancodazen) [reply beta-src/src/components/svgr-components/WDActionIcon.tsx:12]: Yup, thanks for reviewing Kenneth

Comment (bryancodazen) [reply beta-src/src/components/svgr-components/WDActionIcon.tsx:12]: Ill remove

**Pull request merged**